WalletWasabi.Fluent/ViewModels/Dialogs/ShowQrCameraDialogViewModel.cs

```diff
@@ -45,7 +46,11 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 
 		Observable.FromEventPattern<string>(_qrReader, nameof(_qrReader.InvalidAddressFound))
 			.ObserveOn(RxApp.MainThreadScheduler)
-			.Subscribe(args => Message = $"Invalid QR code.")
+			.Subscribe(args =>
+			{
+				ErrorMessage = "The scanned Bitcoin address is not valid";
```

Comment:
What if the user scans something else? How about:

```suggestion
				ErrorMessage = "The scanned QR code is not a valid Bitcoin address.";
```